packages/terra-clinical-header/src/themes/orion-fusion-theme/Header.module.scss

```diff
@@ -0,0 +1,34 @@
+:local {
+  .orion-fusion-theme {
+    // .flex-header
+    --terra-clinical-header-align-items: center;
+    --terra-clinical-header-background-color: #f4f4f4;
+    --terra-clinical-header-border-bottom: 0.16666667rem solid #64696c;
+    --terra-clinical-header-color: #1c1f21;
+    --terra-clinical-header-min-height: 2.83333rem;
+    --terra-clinical-header-padding: 0;
+
+    // .flex-collapse
+    --terra-clinical-header-flex-collapse-align-self: center;
+    --terra-clinical-header-flex-collapse-margin: 0.5rem;
+
+    // .subheader
+    --terra-clinical-header-subheader-background-color: #f4f4f4;
+    --terra-clinical-header-subheader-min-height: 2.83333rem;
```

Comment:
@jeremyfuksa I guess on this value. By not providing a value, the default min-height is larger than the regular min-height value for this theme. Can you validate or provide an updated value for this?

Reply:
Since there's no actual Fusion equivalent to a subheader for this, I think keeping the two types at the same height is OK for now. If we get a new design direction from Seth down the line, we can address then.

Reply:
sounds good. I will update the value to match